### PR TITLE
feat: socials share

### LINKED ISSUE #11: SocialsShare component
#### Screenshots
Screenshot 2020-03-15 at 11 59 37

#### Used by
 - Ce Ma Fac
 - Stam Acasa
 - Diaspora Hub

#### Name
 - SocialsShare (I think we could find a better name for this)

#### Props `SocialsShare`
- currentPage (`string`)

#### Other
 - Is the only prop the current page that we want to share?

### PR DESCRIPTION
**Pull request by adascaliteiradu**
Closes #11 

* Socials share component
* Knobs addon for storybook

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": false,
   "dependencies": {
-    "bulma": "^0.8.0"
+    "@storybook/addon-knobs": "^5.3.17",
```
Comment (Utwo) [inline package.json]: We should put this in devDependencies

File: src/components/social-share/social-share.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.__social-share-container {
+  display: flex;
+  align-items: center;
+  & > * {
+    padding: 4px !important;
```
Comment (moonflare) [inline src/components/social-share/social-share.scss]: We can use `margin` and avoid using `!important`
```suggestion
    margin: 4px;
```

File: src/components/social-share/social-share.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.__social-share-container {
```
Comment (moonflare) [inline src/components/social-share/social-share.scss]: Any reason we prefix this with ` .__`? Maybe something like this?
```suggestion
.socials-share-container {
```

Comment (moonflare) [reply src/components/social-share/social-share.scss]: Also, in the design we have a `1px border-bottom`, can you please add that one as well?

Comment (adascaliteiradu) [reply src/components/social-share/social-share.scss]: To avoid style collision.

Comment (moonflare) [reply src/components/social-share/social-share.scss]: Makes sense. Can you please add the `border` tho? 😄

**Pull request merged**